test/models/charge-version-note.model.test.js

```diff
@@ -0,0 +1,92 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Test helpers
+const ChargeVersionHelper = require('../support/helpers/charge-version.helper.js')
+const ChargeVersionModel = require('../../app/models/charge-version.model.js')
+const ChargeVersionNoteHelper = require('../support/helpers/charge-version-note.helper.js')
+const UserHelper = require('../support/helpers/user.helper.js')
+const UserModel = require('../../app/models/user.model.js')
+
+// Thing under test
+const ChargeVersionNoteModel = require('../../app/models/charge-version-note.model.js')
+
+describe('Charge Version Note model', () => {
+  let testRecord
+
+  beforeEach(async () => {
+    testRecord = await ChargeVersionNoteHelper.add()
+  })
+
+  describe('Basic query', () => {
+    it('can successfully run a basic query', async () => {
+      const result = await ChargeVersionNoteModel.query()
+        .findById(testRecord.id)
+
+      expect(result).to.be.an.instanceOf(ChargeVersionNoteModel)
+      expect(result.id).to.be.equal(testRecord.id)
+    })
+  })
+
+  describe('Relationships', () => {
+    describe('when linking to charge version', () => {
+      let testChargeVersion
+
+      beforeEach(async () => {
+        testChargeVersion = await ChargeVersionHelper.add({ noteId: testRecord.id })
+      })
+
+      it('can successfully run a related query', async () => {
+        const query = await ChargeVersionNoteModel.query()
+          .innerJoinRelated('chargeVersion')
+
+        expect(query).to.exist()
+      })
+
+      it('can eager load the charge version', async () => {
+        const result = await ChargeVersionNoteModel.query()
+          .findById(testRecord.id)
+          .withGraphFetched('chargeVersion')
+
+        expect(result).to.be.instanceOf(ChargeVersionNoteModel)
+        expect(result.id).to.equal(testRecord.id)
+
+        expect(result.chargeVersion).to.be.instanceOf(ChargeVersionModel)
+        expect(result.chargeVersion).to.equal(testChargeVersion)
+      })
+    })
+
+    describe('when linking to user', () => {
+      let testUser
+
+      beforeEach(async () => {
+        testUser = await UserHelper.add({ id: testRecord.userId })
+      })
```

Comment:
We should be able to avoid creating a new user record by instead `select()` an existing one. Shout if you need any help/extra info about how to go about this.

Reply:
Ahh completely forgot about that, implemented it now 👍🏾